python/ray/tune/BUILD

```diff
@@ -474,6 +474,16 @@ py_test(
 #     args = ["--smoke-test"]
 # )
 
+py_test(
+    name = "pbt_transformers_example",
+    size = "medium",
+    srcs = ["examples/pbt_transformers/pbt_transformers.py"],
```

Comment:
I think you need to add `transformers` into the right requirements txt file? 

And perhaps also the documentation txt file?

Reply:
Added transformers to tune dependencies.

Reply:
Why would I need to add transformers to the requirements-doc.txt? Or are you referring to something else?